### PR TITLE
Fix UI testing for new openshift-console

### PR DESCRIPTION
**Pull request by petr-balogh**
Fixes: #2608

Signed-off-by: Petr Balogh <pbalogh@redhat.com>

### CODE REVIEW

**Comment by petr-balogh** [comment]
Verified here: https://ocs4-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/job/qe-deploy-ocs-cluster/10681/console

The openshift-console testing were executed , but failed on some test issue which is not part of this PR.

**Comment by petr-balogh** [comment]
Another verification:
https://ocs4-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/job/qe-deploy-ocs-cluster/10684/console

File: conf/ocsci/openshift-console/openshift_console_no_failfast.yaml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+---
+# To avoid skipping remaining portion of tests upon encountering the first failure
+RUN:
+  openshift_console_no_failfast: true
```
Comment (vasukulkarni) [inline conf/ocsci/openshift-console/openshift_console_no_failfast.yaml:4]: s/openshift_console_no_failfast/openshift_console_failfast/  we dont have to negate a boolean variable.

Comment (petr-balogh) [reply conf/ocsci/openshift-console/openshift_console_no_failfast.yaml:4]: Variable used in openshift console has exact this name: NO_FAILFAST so I am keeping their convention.

I think that openshift console test execution fails after first test failed. Wit this env variable set, the openshift console will continue with other tests. @ebondare asked me to allow this option and I've created this configuration file for it.

@ebondare am I right?

Comment (ebondare) [reply conf/ocsci/openshift-console/openshift_console_no_failfast.yaml:4]: @petr-balogh yes, that's true, thank you

**Pull request merged**